crates/nu-command/src/commands/str_/to_integer.rs

```diff
@@ -29,7 +28,7 @@ impl WholeStreamCommand for SubCommand {
         Signature::build("str to-int")
             .named("radix", SyntaxShape::Number, "radix of integer", Some('r'))
             .rest(
-                SyntaxShape::ColumnPath,
+                SyntaxShape::Any,
```

Comment:
Why switch to Any here?

Reply:
Oops, from another PR. Basically it allows to use `str to-int table_of_columns` so you can do either:

```
... | str to-int col1 col2 col3
... | str to-int $(= some_table)
```

Reply:
This is related to the issue/concern I brought up in the call the other day.

Reply:
Would you mind if we did that as a separate change?

Reply:
No problem.